[PATCH] `./configure --disable-systemd` fails with missing systemd.

When I run `./configure --disable-systemd`, it is expected to not complain about missing systemd. But it does:

```
checking for DEPS... no
configure: error: Package requirements (libudev libsystemd > 219) were not met:

Package 'libsystemd', required by 'virtual:world', not found
```

The actual parameter which has to be passed to `./configure` seems to be `--enable-disable-systemd`, but `./configure --help` says it is `--disable-systemd`.

Fix attached: [`configure-fix-disable-systemd.patch.txt`](https://github.com/albfan/miraclecast/files/13220743/configure-fix-disable-systemd.patch.txt).


Regards!

> Probably better:
>
> \[&hellip;\]
> 
> can you test again with:
> 
> ```
> ./configure --disable-systemd
> ```

I applied your patch and tested, now `./configure --disable-systemd` runs through.

_This_ issue can now be closed. (But running `make` still fails with  
```
rtsp.c:31:10: fatal error: systemd/sd-event.h: No such file or directory
   31 | #include <systemd/sd-event.h>
      |          ^~~~~~~~~~~~~~~~~~~~
```
so [the other patch](https://aur.archlinux.org/cgit/aur.git/tree/miraclecast-use-elogind.patch?h=miraclecast-nosystemd-git) is still needed.)

--- a/configure.ac
+++ b/configure.ac
@@ -45,8 +45,15 @@ AC_ARG_VAR(IP_BINARY, [Path for ip binary])
 if test -z "$IP_BINARY"; then
 	IP_BINARY=/bin/ip
 fi
-AC_ARG_ENABLE([disable-systemd],
-              AS_HELP_STRING([--disable-systemd], [Disable systemd]), [], [use_libsystemd=yes], [Disable systemd using elogind])
+AC_ARG_ENABLE([systemd],
+              AS_HELP_STRING([--disable-systemd], [Disable systemd]),
+              [case "${enableval}" in
+                yes) use_libsystemd=yes ;;
+                no)  use_libsystemd=no ;;
+                *) AC_MSG_ERROR([bad value ${enableval} for --enable-systemd]) ;;
+              esac],
+              [use_libsystemd=yes])
+
 AC_DEFINE_UNQUOTED([IP_BINARY], [$IP_BINARY], [Path for ip binary])
 
 #

